arangod/RestHandler/RestAdminLogHandler.cpp

```diff
@@ -425,8 +425,15 @@ void RestAdminLogHandler::handleLogLevel() {
     if (slice.isString()) {
       Logger::setLogLevel(slice.copyString());
     } else if (slice.isObject()) {
+      if (VPackSlice all = slice.get("all"); all.isString()) {
```

Comment:
can we have a constant for that?

Reply:
Yes perhaps`VPackSlice const all` as doesn't have its value changed somewhere else?

Reply:
No, I meant a constant for "all" topic name